portfolio/src/main/webapp/index.html

```diff
@@ -22,11 +22,36 @@
       </div>
       
       <section id="intro">
-        <h1 class="big-header-main">Hello, I'm <span style="color: #9366e3">Sabrina Snider</span>.</h1>
-        <h2 class="sub-header-main">I like writing code.</h2>
-        <a href="#work-section" id="work-button" class="white-button">My Work</a>
+        <div id="intro-content">
+          <div>
+            <div id="intro-text">
+              <h2 class="sub-header-main">Hello World</h2>
+              <h1 class="big-header-main">I'm <span style="color: #9366e3; font-weight: bold;">Sabrina</span> and I like to code</h1>
```

Comment:
Move the inline styles for your the span into the style sheet and either add a class or id to the span.